core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -770,6 +770,12 @@ bool core_mmu_user_mapping_is_active(void)
 #ifdef ARM32
 void core_mmu_get_user_map(struct core_mmu_user_map *map)
 {
+	/*
+	 * Foreign interrupts must be disabled before playing with current
+	 * core since we otherwise may be rescheduled to a different core.
+	 */
+	assert(thread_get_exceptions() & THREAD_EXCP_FOREIGN_INTR);
```

Comment:
Not needed now that `get_core_pos()` has the assert

Reply:
right :)